cli/ccom.go

```diff
@@ -33,13 +33,13 @@ func main() {
 				Usage:   "Benchmark the specified compiler executable and run it n times",
 			},
 			&cli.StringFlag{
-				Name:    "block-comment-chars-open",
-				Aliases: []string{"bcco"},
+				Name:    "block-comment-iden-open",
+				Aliases: []string{"bcio"},
 				Usage:   "Specifies the opening block comment character(s) of your data format",
```

Comment:
change to `identifier`s (note the multiple occurrences in this file)

Reply:
Don't now what you mean, sorry. What's wrong with that?

Reply:
`iden` -> `identifier`?

Reply:
I mean the value for the `Usage` key: "Specifies the opening block comment character(s) of your data format"
You've replaced "chars" by "iden" (line 36+37), so why not replace "characters" by "iden" or "identifier" as well...

Reply:
Ahh okay. Know what you mean. Gonna fix that real quick ...

Reply:
Should be fine now.